WordPress/WordPressNotificationContentExtension/Sources/NotificationViewController.swift

```diff
@@ -2,11 +2,66 @@ import UIKit
 import UserNotifications
 import UserNotificationsUI
 
-class NotificationViewController: UIViewController, UNNotificationContentExtension {
+// MARK: - NotificationViewController
 
-    @IBOutlet var label: UILabel?
+/// Responsible for enhancing the visual appearance of designated push notifications.
+@objc(NotificationViewController)
+class NotificationViewController: UIViewController {
 
+    // MARK: Properties
+
+    /// Long Look content is inset from iOS container bounds
+    private struct Metrics {
+        static let verticalInset = CGFloat(20)
+    }
+
+    /// This view model contains the attributes necessary to render a rich notification
+    private var viewModel: RichNotificationViewModel? {
+        didSet {
+            setupContentView()
+        }
+    }
+
+    /// The subview responsible for rendering a notification's Long Look
+    private var contentView: NotificationContentView?
+
+    // MARK: UIViewController
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+
+        _ = view.subviews.map { $0.removeFromSuperview() }
```

Comment:
Not a big difference, but we could do `view.subviews.forEach { $0.removeFromSuperview() }` here instead.